static/src/components/markdown-editor/html-editor.css

```diff
@@ -0,0 +1,19 @@
+:host {
+  display: block;
+  width: 100%;
+}
+
+.editor-content {
+  min-height: 1.5em;
+  padding-bottom: var(--ion-padding, 0);
+  outline: none;
+}
+
+:host:focus-within ion-label {
+  color: var(--ion-color-primary);
+}
+
+ion-buttons {
+  position: sticky;
```

Comment:
This doesn't appear to be working. Do I need to set to  a larger number to avoid the header toolbar?